virtual_machine_config.go

```diff
@@ -114,3 +114,24 @@ func (vmc *VirtualMachineConfig) MergeIPConfigs() map[string]string {
 	}
 	return vmc.IPConfigs
 }
+
+func (vmc *VirtualMachineConfig) MergeDisks() map[string]string {
+	mergedDisks := make(map[string]string)
+
+	for k, v := range vmc.MergeIDEs() {
```

Comment:
not sure i know the data for all disk types, no chance of this key getting clobbered?

Reply:
There should not be any conflict between those keys as PVE sends all VM options as a flat JSON map. Disks are all prefixed with the relevant emulated hardware type (`virtioX`, `scsiX`, `ideX`, etc).

Reply:
<img width="613" alt="Screenshot 2024-06-04 at 16 41 40" src="https://github.com/luthermonson/go-proxmox/assets/80721144/1faac062-0601-4684-8eb7-5ae47aba0270">

I cover those formats through the Disk configuration of the Virtual Machine. I'm thinking like @jqueuniet for the data format according to the API documentation here. I'm just trying to sum those in a single map that I can use.